src/curated-corpus/components/ScheduleSummaryCard/ScheduleSummaryCard.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+
+export type ScheduleSummary = {
+  name: string;
+  count: number;
+};
+
+interface ScheduleSummaryCardProps {
+  headingCopy: string;
+  items: ScheduleSummary[];
+}
+
+/**
+ * Display a summary of scheduled stories for a given grouping
+ * (for example, topic or publisher).
+ *
+ * @param props
+ * @constructor
+ */
+export const ScheduleSummaryCard: React.FC<ScheduleSummaryCardProps> = (
+  props
+): JSX.Element => {
+  const { headingCopy, items } = props;
+
+  return (
+    <>
+      <TableContainer component={Paper}>
+        <Table size="small">
+          <TableHead>
+            <TableRow>
+              <TableCell>{headingCopy}</TableCell>
+              <TableCell align="right">#</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {items.map((item) => {
+              return (
+                <React.Fragment key={item.name}>
```

Comment:
We could replace `<React.Fragment>` with `<>` and add the `key` prop to the `<TableRow>`?

Reply:
Thanks @Herraj, that was a good catch! Fragment is not needed here at all - I think it was left over after experimenting with other markup (not a table).